client/normal/user.js

```diff
@@ -0,0 +1,12 @@
+import { Template } from 'meteor/templating';
+import { users } from '../../lib/database/user.js';
+//var user = Meteor.subscribe('users');
+Meteor.subscribe('users');
+Template.user.helpers({
+		UserName() {
+	 	return  Meteor.user().services.github.username;
```

Comment:
This will fail for any user that didn't sign up using Github.

Reply:
#44 i didn't try out, as for facebook,i thought it was resolved

Reply:
ok cool, i will put up for facebook too

Reply:
ok i will put up for facebook too

Reply:
Facebook is already solved.

Reply:
Is it possible to change the code acocrdingly and submit another PR for github issue?

Reply:
Well you could, but this is not at all the simplest approach to solving the issue in #44. In fact it doesn't even solve the issue at all unless you also go through the entire codebase and check every place where usernames are used and then edit those parts as well. The simplest approach to use is the one that was already used for facebook.

Reply:
So, is it better to tackle another issue for now? and properly understand the codebase for this PR.

Reply:
This is already one of the most simple issues to solve which is why it's left open as an assessment. If you look at the discussion thread and pull requests associated with this issue you can see that only a few lines of code are needed if you take the right approach. You are welcome to continue working on this as an open source project to build up your experience and public profile, but what we are looking for right now in terms of _paid_ contributors are people who have bit more experience with Meteor.